libraries/helpers.rb

```diff
@@ -57,7 +57,7 @@ def check_from_resource
       spec['runtime_assets'] = new_resource.runtime_assets if new_resource.runtime_assets
       spec['secrets'] = new_resource.secrets if new_resource.secrets
       spec['stdin'] = new_resource.stdin
-      spec['subdue'] = new_resource.subdue if new_resource.subdue
+      spec['subdues'] = new_resource.subdues if new_resource.subdues
```

Comment:
are both valid?

Reply:
In my opinion no, but it's unclear how the folks at Sensu intended to resolve this as the original subdue implementation still appears as null in the CheckConfig reference.  See the YML output from their [cron example check](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/checks/#example-cron-checks) vs the actual [subdues documentation section](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/checks/#subdues).


In practice one of my example checks using this new functionality lists both but I'm unable to set the original subdue as anything other than null:

```
type: Check
api_version: core/v2
metadata:
  created_by: s_sensu
  labels:
    sensu.io/managed_by: sensuctl
  name: check_process
  namespace: default
spec:
  check_hooks: null
  command: check_process.rb
  env_vars: null
  executed: 0
  handlers:
  high_flap_threshold: 0
  history: null
  interval: 15
  is_silenced: false
  issued: 0
  last_ok: 0
  low_flap_threshold: 0
  occurrences: 0
  occurrences_watermark: 0
  output: ""
  output_metric_format: ""
  output_metric_handlers: null
  pipelines: []
  proxy_entity_name: ""
  publish: true
  round_robin: false
  runtime_assets:
  - runtime_ruby
  scheduler: ""
  secrets: null
  status: 0
  stdin: false
  subdue: null
  subdues:
  - begin: "2022-04-18T16:00:00-05:00"
    end: "2022-04-18T17:00:00-05:00"
    repeat:
    - weekdays
  - begin: "2022-04-22T16:00:00-05:00"
    end: "2022-04-22T17:00:00-05:00"
    repeat:
    - fridays
  - begin: "2022-04-23T00:00:00-05:00"
    end: "2022-04-23T23:59:59-05:00"
    repeat:
    - saturdays
  - begin: "2022-04-24T00:00:00-05:00"
    end: "2022-04-24T17:00:00-05:00"
    repeat:
    - sundays
  subscriptions:
  - check_process
  timeout: 0
  total_state_change: 0
  ttl: 0
```